README.md

```diff
@@ -25,33 +31,119 @@ The package is a console application that can be used to perform common tasks in
 The package could be installed with [Composer](https://getcomposer.org):
 
 ```shell
-composer create-project --stability=dev yiisoft/app-console <your project>
+composer create-project yiisoft/app-console <your project>
 ```
 
 ## General usage
 
-### Create command console
+Console is available as `./yii` from the root directory of the application:
+
+```shell
+$ ./yii
+
+Yii Console 1.0
+
+Usage:
+  command [options] [arguments]
+
+Options:
+  -h, --help            Display help for the given command. When no command is given display help for the list command
+  -q, --quiet           Do not output any message
+  -V, --version         Display this application version
+      --ansi|--no-ansi  Force (or disable --no-ansi) ANSI output
+  -n, --no-interaction  Do not ask any interactive question
+      --config=CONFIG   Set alternative configuration name
+  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
+
+Available commands:
+  completion  Dump the shell completion script
+  echo        An example command that echoes exactly what it is told to.
+  help        Display help for a command
+  list        List commands
+  serve       Runs PHP built-in web server
+```
+
+Help for specific command could be displayed by adding `--help` to the command itself:
+
+```shell
+$ ./yii echo --help
+
+Description:
+  An example command that echoes exactly what it is told to.
+
+Usage:
+  echo [<sentence>]
+
+Arguments:
+  sentence              Sentence to say. [default: "Hello!"]
+
+Options:
+  -h, --help            Display help for the given command. When no command is given display help for the list command
+  -q, --quiet           Do not output any message
+  -V, --version         Display this application version
+      --ansi|--no-ansi  Force (or disable --no-ansi) ANSI output
+  -n, --no-interaction  Do not ask any interactive question
+      --config=CONFIG   Set alternative configuration name
+  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
+```
+
+Using the command is like the following: 
+
+```shell
+$ ./yii echo
+You said: Hello!
+
+$ ./yii echo 'Code something'
+You said: Code something
+```
+
+## Environments
+
+Out of the box, three environments are available:
```

Comment:
Need to remove yiisoft/yii-debug params from configration

Reply:
Done.